puppet/modules/profiles/manifests/base/monitoring.pp

```diff
@@ -1,4 +1,4 @@
-class profiles::monitoring::client (
+class profiles::base::monitoring (
   String[1] $prometheus_username,
   String[1] $prometheus_password,
   Stdlib::HTTPUrl $prometheus_url,
```

Comment:
these are non-optional (as otherwise there is no endpoint to report to), but are undefined as I kept them "secret", which makes the tests fail now.
1. I could provide dummy values in `internal.yaml`, but that seems like a hack.
1. I could introduce a dummy hiera file for rspec only
1. I could make these actually optional (and not deploy the `remote_write` config if they are unset)

Preferences?

Reply:
I went with 3